Jenkinsfile

```diff
@@ -256,3 +315,15 @@ def uploadInstaller(String platform) {
         echo "Skipped upload for branch ${env.BRANCH_NAME}"
     }
 }
+
+def linkInstaller(String platform, String installer, String extension) {
+    if (env.BRANCH_NAME == releaseBranch) {
+        def packageJSON = readJSON file: "package.json"
+        String version = "${packageJSON.version}"
+        sshagent(['projects-storage.eclipse.org-bot-ssh']) {
+            sh "ssh genie.theia@projects-storage.eclipse.org ln -s /home/data/httpd/download.eclipse.org/theia/latest/${platform}/${installer}.${extension} /home/data/httpd/download.eclipse.org/theia/latest/${platform}/${installer}-${version}.${extension}"
```

Comment:
Question: I do not see anywhere, that the package is renamed to have the version number in it. I assume that's the final name for the package, that gets uploaded to `eclipse.org`?

Reply:
Ah, that's what the symlink is for I think?